src/components/Sidebar/SidebarMenuItem.tsx

```diff
@@ -1,37 +1,50 @@
 import { SvgIconComponent } from '@mui/icons-material';
-import { ListItem, ListItemButton, ListItemIcon, ListItemText } from '@mui/material';
+import { ListItem, ListItemButton, ListItemIcon, ListItemText, useTheme } from '@mui/material';
 import React from 'react';
-import { useNavigate } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
 import { ISidebarMenuItem } from 'models/SidebarMenuItem';
 
 type Props = {
   collapsed: boolean;
   item: ISidebarMenuItem;
-  selected?: boolean;
 };
 
 function renderIcon(Icon: SvgIconComponent | (() => JSX.Element)) {
   return <Icon></Icon>;
 }
 
-export const SidebarMenuItem = ({ collapsed, item, selected = false }: Props) => {
-  const navigate = useNavigate();
+export const SidebarMenuItem = ({ collapsed, item }: Props) => {
+  const theme = useTheme();
+
   return (
-    <ListItem disablePadding>
-      <ListItemButton
-        id={`side-item-${item.label.toLowerCase()}`}
-        onClick={() => {
-          navigate(item.route);
-        }}
-        selected={selected}
-        sx={{ px: 3 }}>
-        {item.icon && <ListItemIcon>{renderIcon(item.icon)}</ListItemIcon>}
-        {!collapsed && (
-          <ListItemText
-            id={`menu-item-${item.label.toLowerCase()}`}
-            primary={item.label}></ListItemText>
-        )}
-      </ListItemButton>
-    </ListItem>
+    <NavLink
```

Comment:
Please note: `NavLink` is rendered as `a` tag, while `ListItem` as `li`.
So, referring to the [spec](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/nav) , you've to nest `NavLink`  *in* the `ListItem`. I suppose you've to replace `ListItemButton` with `NavLink`. 
Read carefully the doc, at the final render we need to have a structure like this:

```
<nav>
  <ol>
    <li><a>Label</a>
    </li>
    <li><a>Label</a>
    </li>
  </ol>
</nav>
```
Enforce accessibility with these [tips](https://www.w3.org/WAI/tutorials/menus/structure/)